### PR TITLE
Move slates.nim here

### PR DESCRIPTION
**Pull request by guzba**

### CODE REVIEW

File: src/slates.nim [hunk 1]
```diff
@@ -0,0 +1,61 @@
+import flippy, snappy, streams, strformat
```
Comment (treeform) [inline src/slates.nim:1]: Can you add snappy to the nimble req.

File: src/slates.nim [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+type Slate* = object
+  mipmaps*: seq[Image]
+
+func width*(slate: Slate): int =
+  slate.mipmaps[0].width
+
+func height*(slate: Slate): int =
+  slate.mipmaps[0].height
+
+proc save*(slate: Slate, filePath: string) =
```
Comment (treeform) [inline src/slates.nim:12]: Can you add a simple slate load/save test?

File: src/slates.nim (hunk 1 above)
Comment (guzba) [reply src/slates.nim:1]: Yep, good catch. Done.

File: src/slates.nim (hunk 2 above)
Comment (guzba) [reply src/slates.nim:12]: Sure, done.

**Pull request merged**